readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -265,6 +265,43 @@ def test_project_features_multiple_projects(self):
         self.assertIn('features', resp.data)
         self.assertEqual(resp.data['features'], [feature.feature_id])
 
+    def test_project_pagination(self):
+        for _ in range(100):
+            get(Project)
+
+        resp = self.client.get('/api/v2/project/')
```

Comment:
I think you can get the url by using `reverse`. So it will be easier to manage in the future